mobify/progressive-web-scaffold
# Move customer entity id to the checkout store

This PR moves the customer entity ID from UI -> checkoutShipping to the checkout store since it's needed for multiple pages in the checkout. 

 **JIRA**: n/a
 **Linked PRs**: n/a

## Changes
- Moved customerEntityID into the checkout store
- Added new selector for customerEntityID 
- Updated existing references to the customerEntityID selector to use the new one

## How to test-drive this PR
- (necessary config changes)
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart
- navigate to: https://www.merlinspotions.com/checkout
- confirm you're still able to submit the shipping information
